src/components/application_manager/include/application_manager/command_factory.h

```diff
@@ -54,6 +58,101 @@ class CommandFactory {
   virtual CommandSharedPtr CreateCommand(
       const commands::MessageSharedPtr& message,
       commands::Command::CommandSource source) = 0;
+  virtual bool IsAbleToProcess(
```

Comment:
@BSolonenko please give doxygen comments to functions

Reply:
@LuxoftAKutsan gave in https://github.com/LuxoftAKutsan/sdl_core/pull/275/commits/75224d1cd9b7d4fc21af67229e902ff04fdaedb9